guesscard-index.html

```diff
@@ -12,185 +12,189 @@
     <h1>Guess the Card</h1> <!--Heading with my name-->
     
     <p>A card game, where the user has to guess them</p>
+    <!--Div container to hold the grid of 36 cards-->
     <div class="container">
+
+    <!--Div to hold each image.
+    Used two divs might be needed later. So leaving it for now-->
       <div class="cell cell-1">
-        <div class="check">
-          <img src="./res/images/card-background.png" class="cell__img">
+        <div>
+          <img class="cell__img" id=cell__img1 src="">
         </div>
       </div>
       <div class="cell cell-2">
```

Comment:
Mhhh this whole cell thing looks somehow like repetition, how could we do that in a Javascript way?

Reply:
Maybe a loop could help us here?

Reply:
implemented looping from javascript, now the divs and img are created from script